### PR TITLE
Add support for multiple phantomjs-options

### PR DESCRIPTION
**Pull request by jasonjensen**
All tests do not pass, but neither do they before the changes.

### CODE REVIEW

**Comment by jonkemp** [comment]
Is this code change related to this issue?

https://github.com/jonkemp/node-qunit-phantomjs/issues/23

**Comment by jasonjensen** [comment]
Yes. Sorry, I couldn't find that issue for some reason.

File: index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = function (filepath, options, callback) {
     }
 
     if (opt['phantomjs-options'] && opt['phantomjs-options'].length) {
-        childArgs.push(opt['phantomjs-options']);
+        childArgs = childArgs.concat(opt['phantomjs-options']);
```
Comment (jonkemp) [inline index.js]: This is what is needed but I would suggest using the method that gulp-qunit does.
```js
        if (options['phantomjs-options'] && options['phantomjs-options'].length) {
            if (Array.isArray(options['phantomjs-options'])) {
                childArgs = childArgs.concat(options['phantomjs-options']);
            } else {
                childArgs.push(options['phantomjs-options']);
            }
        }
```

Comment (jonkemp) [reply index.js]: You can see the change in gulp-qunit when it was made here as well as the included test.

https://github.com/jonkemp/gulp-qunit/pull/30/files

Please include the test for this as well.

**Comment by jonkemp** [comment]
nm I'll just do it

**Pull request merged**